### PR TITLE
Adding Dynamic Templates content to Email Testing page + release note

### PR DESCRIPTION
**Pull request by Whatthefoxsays**
**Description of the change**:
**Reason for the change**:
**Link to original source**:

Closes #

### CODE REVIEW

File: content/docs/ui/sending-email/email-testing.md [hunk 1]
```diff
@@ -47,6 +42,8 @@ If you find yourself in need of more credits than your Marketing Campaigns plan
 
 ## Running an Email Test
 
+Email testing is available for both Marketing Campaigns and emails utilizing Dynamic Templates.
```
Comment (lweinig) [inline content/docs/ui/sending-email/email-testing.md]: can you add "within Email API plans" to this sentence?

File: content/docs/release-notes/2019-19-12.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: "release notes 2019-19-12"
+date: 2019-19-12
+releaseType:
+  - ui
+---
+
+### Email Content Testing for Dynamic Templates
+
+Test your email content against spam filters, validate hyperlinks, and preview HTML rendering across browsers and devices with Email Testing for Dynamic Templates. Monthly testing credits available for Trial, Pro and Premier plans. See our Email Testing [documentation](https://sendgrid.com/docs/ui/sending-email/email-testing/) for more details.
```
Comment (lweinig) [inline content/docs/release-notes/2019-19-12.md]: Monthly testing credits included for Trial, Pro and Premier plans, with additional credits available for purchase.

File: content/docs/ui/sending-email/email-testing.md [hunk 3]
```diff
@@ -11,14 +11,7 @@ seo:
   override: true
   Description: How to use SendGrid's new email testing feature to see how your email shows up in different inboxes and across different devices and operating systems.
```
Comment (lweinig) [inline content/docs/ui/sending-email/email-testing.md]: How to use SendGrid's new email testing feature to preview your email renders, performs against spam filters, and validate the links within your email.

**Comment by thinkingserious** [comment]
Hello @Whatthefoxsays,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**